scripts/reset_data_package_cache.py

```diff
@@ -48,7 +49,7 @@ def update_column_type_metadata(bucket: str, client):
             output[study][data_package][version]["last_data_update"] = (
                 resource["LastModified"].now().isoformat()
             )
-            output[study][data_package][version]["s3_path"] = resource["Key"]
+            output[study][data_package][version]["s3_path"] = f"s3://{bucket}/{resource['Key']}"
```

Comment:
in doing some debugging, this migration had been assigning keys to what was specified as a path, which generated some inconsistencies depending on how the cache was built. There's some errant downstream changes here and there as a result of this change.

Longer term it might be better to rework this to store less logic directly here, so there's not a natural split? but at least actually adhering to the name of the thing should be helpful.